WORKSPACE

```diff
@@ -131,6 +131,50 @@ rbe_autoconfig(
     use_checked_in_confs = False,
 )
 
+# RBE Autoconfig targets to do integration testing on the automatic toolchain
+# configs release process.
+load(
+    "//tests/config/dependency-tracking:ubuntu1604_no_updates.bzl",
+    _ubuntu1604_bazel_no_updates = "bazel",
+    _ubuntu1604_configs_version_no_updates = "configs_version",
+    _ubuntu1604_digest_no_updates = "digest",
+    _ubuntu1604_registry_no_updates = "registry",
+    _ubuntu1604_repository_no_updates = "repository",
+)
+
+# Automatic E2E test config generation target for RBE Ubuntu 16.04 that should
+# not generate any new configs.
+rbe_autoconfig(
+    name = "rbe_ubuntu1604_configs_no_update_test",
+    bazel_version = _ubuntu1604_bazel_no_updates,
+    digest = _ubuntu1604_digest_no_updates,
+    output_base = "configs/ubuntu16_04_clang/{}".format(_ubuntu1604_configs_version_no_updates),
+    registry = _ubuntu1604_registry_no_updates,
+    repository = _ubuntu1604_repository_no_updates,
+    use_checked_in_confs = False,
+)
+
+load(
+    "//tests/config/dependency-tracking:ubuntu1604_updates_exist.bzl",
+    _ubuntu1604_bazel_updates_exist = "bazel",
```

Comment:
name this 
_bazel_trigger_config_gen
_digest_trigger_config_gen

Also name the file 
//tests/config/dependency-tracking:trigger_config_gen.bzl

Similar for the one above, file should be
//tests/config/dependency-tracking:no_updates.bzl

Reply:
Done